deployments/testnet/tokenBridgeVAAs.csv

```diff
@@ -9,8 +9,8 @@ Oasis (7) Testnet Token Bridge,01000000000100eb109d6208ea838e9ae9ef091f4dc4a4d66
 Algorand (8) Testnet Token Bridge,010000000001006e90fe2a505f0d67b6ca6221a42f55957bf59201451ffbecbfa4a3416a8491c45e8bcaa18a3c422f5548d98db788f89ac6c8c89d1fe6334860baccc8fedb7c3400626a9eed41c71ee1000100000000000000000000000000000000000000000000000000000000000000040000000041c71ee120000000000000000000000000000000000000000000546f6b656e42726964676501000000086241ffdc032b693bfb8544858f0403dec86f2e1720af9f34f8d65fe574b6238c
 Aurora (9) (NEAR) Testnet Token Bridge,0100000000010096e79a229cdd3717f0b47bf7fb65b88202c4ec5fd9cc7d5f319cbe27b0f57d0c2292204c9df25e060e7e9632a54fefca88f1ca95c76ca083a07a68d59de9964701000000010000000100010000000000000000000000000000000000000000000000000000000000000004000000000227003a00000000000000000000000000000000000000000000546f6b656e4272696467650100000009000000000000000000000000d05ed3ad637b890d68a854d607eeaf11af456fba
 Fantom (10) Testnet Token Bridge,010000000001007bc5bdc2253a6deec065a6e573f7bdd363383b91210054d8ce7558d9b8185a60157370f56f35d03fc434ecf90fc652cf31b9b4a2a15eb43c5ad8a664356ef81e00000000010000000100010000000000000000000000000000000000000000000000000000000000000004000000000351c06c00000000000000000000000000000000000000000000546f6b656e427269646765010000000a000000000000000000000000599cea2204b4faecd584ab1f2b6aca137a0afbe8
-Karura (11) Testnet Token Bridge,01000000000100c01c50054de5f481bb5cd8b4e5d906d86c7307ba969a0f9f042e410b859528387040360954dc4fd4834d4f68dfea9067f3da3bb4af127f61e3ae04f7d216fc300100000001000000010001000000000000000000000000000000000000000000000000000000000000000400000000026bcc5b00000000000000000000000000000000000000000000546f6b656e427269646765010000000b000000000000000000000000d11de1f930ea1f7dd0290fe3a2e35b9c91aefb37
-Acala (12) Testnet Token Bridge,010000000001006b565b943c90f3634b8352f9089e3b3a0a50e6f32edea47b91b5a1c15f190a0317a7ffc9fc76df4ed98eb25effe70c0218bdcb16ff9799d1ed8b33e756ca08990100000001000000010001000000000000000000000000000000000000000000000000000000000000000400000000059f9b1700000000000000000000000000000000000000000000546f6b656e427269646765010000000c000000000000000000000000e157115ef34c93145fec2fe53706846853b07f42
+Karura (11) Testnet Token Bridge,01000000000100175e61ce980c99ed9e85a7519070a736bc418e3218349bfac7360f9699e852373ae786bc5646da9f3c5ce7b4d1fda26df9d9c0fc8262a0bc84bd4d01d33379020100000001000000010001000000000000000000000000000000000000000000000000000000000000000400000000007d92c400000000000000000000000000000000000000000000546f6b656e427269646765010000000b000000000000000000000000e157115ef34c93145fec2fe53706846853b07f42
+Acala (12) Testnet Token Bridge,01000000000100816474735a7358790cd47c5b970157ce7dd1fa76e67f2a2ed792a80eac41c7e0773ef14294429cda77da6e14b47a5490fd4d8da9e2f59fdd14fed41652ef366c010000000100000001000100000000000000000000000000000000000000000000000000000000000000040000000001e9e32200000000000000000000000000000000000000000000546f6b656e427269646765010000000c000000000000000000000000e157115ef34c93145fec2fe53706846853b07f42
```

Comment:
FWIW, these two are the same, just different sequence numbers.
[before](https://wormholescan.io/#/vaa-parser/AQAAAAABAGtWW5Q8kPNjS4NS+QieOzoKUObzLt6ke5G1ocFfGQoDF6f/yfx2307ZjrJe/+cMAhi9yxb/l5nR7Ysz51bKCJkBAAAAAQAAAAEAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAWfmxcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVG9rZW5CcmlkZ2UBAAAADAAAAAAAAAAAAAAAAOFXEV7zTJMUX+wv5TcGhGhTsH9C)
[after](https://wormholescan.io/#/vaa-parser/AQAAAAABAIFkdHNac1h5DNR8W5cBV8590fp25n8qLteSqA6sQcfgdz7xQpRCnNp32m4UtHpUkP1Njani9Z/dFP7UFlLvNmwBAAAAAQAAAAEAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAHp4yIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVG9rZW5CcmlkZ2UBAAAADAAAAAAAAAAAAAAAAOFXEV7zTJMUX+wv5TcGhGhTsH9C)